ray-operator/apis/ray/v1/rayjob_types.go

```diff
@@ -103,7 +107,8 @@ type RayJobSpec struct {
 	EntrypointNumGpus float32 `json:"entrypointNumGpus,omitempty"`
 	// EntrypointResources specifies the custom resources and quantities to reserve for the
 	// entrypoint command.
-	EntrypointResources string `json:"entrypointResources,omitempty"`
+	EntrypointResources string           `json:"entrypointResources,omitempty"`
+	SubmitterConfig     *SubmitterConfig `json:"submitterConfig,omitempty"`
```

Comment:
`submissionConfig` or `jobSubmissionConfig` might be better than `submitterConfig` because we are configuring how the job is being submitted instead of the submitter pod

Reply:
Sync with Andrew offline: We will keep submitterConfig because we always use 'submitter' for the Kubernetes job in RayJob documents.